pkg/processor/test/suite/suite.go

```diff
@@ -252,8 +267,8 @@ func (suite *TestSuite) GetTestFunctionsDir() string {
 // GetTestHost returns the host on which a remote testing entity resides (e.g. brokers, functions)
 func (suite *TestSuite) GetTestHost() string {
 
-	// If an env var is set, use that. otherwise localhost
-	return common.GetEnvOrDefaultString("NUCLIO_TEST_HOST", "localhost")
+	// If an env var is set, use that, otherwise 127.0.0.1
+	return common.GetEnvOrDefaultString("NUCLIO_TEST_HOST", "127.0.0.1")
```

Comment:
why?
suggest to keep localhost actually: https://www.pixelstech.net/article/1538275121-Difference-between-localhost-and-127-0-0-1
Note:
> "It's very important to know that it will not go through network card when using localhost while it will when using 127.0.0.1, this means localhost will not be affected by network card configurations and firewall settings, all ports are open as well"

Reply:
When VPN is on, you won't be able to run tests.

Reply:
I do not understand why exactly. Please explain offline. There was nothing in the code or comment to suggest that this change is now mandatee. (It's been local host and working before this pr)

Reply:
Of you have a clear understanding of why than please feel free to also explain in a comment on the code